CHANGELOG.md

```diff
@@ -54,6 +54,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
     response now contains the ethereum-formatted `Hash` in hex format.
 * (eth) [\#845](https://github.com/cosmos/ethermint/pull/845) The `eth` namespace must be included in the list of API's as default to run the rpc server without error.
 * (evm) [#202](https://github.com/tharsis/ethermint/pull/202) Web3 api `SendTransaction`/`SendRawTransaction` returns ethereum compatible transaction hash, and query api `GetTransaction*` also accept that.
+* (rpc) [#258](https://github.com/tharsis/ethermint/pull/258) Return empty bloomfilter instead of throwing error when it cannot be found (nil or empty).
```

Comment:
```suggestion
* (rpc) [tharsis#258](https://github.com/tharsis/ethermint/pull/258) Return empty `BloomFilter` instead of throwing an error when it cannot be found (`nil` or empty).
```